prime-router/src/main/kotlin/history/DeliveryHistory.kt

```diff
@@ -78,4 +78,41 @@ class DeliveryHistory(
             this.createdAt
         )
     }
+}
+
+/**
+ * Class containing information fetched from the DB when getting delivery facilities
+ *
+ * @property testingLabName the full name of the facility
+ * @property testingLabCity city of the facility
+ * @property testingLabState state of the facility
+ * @property testingLabClia The CLIA number (10-digit alphanumeric) of the facility
+ * @property positive the result (conclusion) of the test. 0 = negative (good usually)
+ * @property countRecords number of facilities included in the object
+ */
+data class DeliveryFacility(
+    val testingLabName: String?,
+    val testingLabCity: String?,
+    val testingLabState: String?,
+    val testingLabClia: String?,
+    val positive: Long?,
+    val countRecords: Long?,
+) {
+    /**
+     * This is a combination of the city and state values
+     * for easier conversion into an output format
+     */
+    val location: String? get() {
+        var loc = this.testingLabCity
+
+        if (this.testingLabState != null) {
+            loc = if (loc != null) {
+                loc + ", " + this.testingLabState
```

Comment:
Check your IntelliJ problems tab.  This will be flagged with the suggestion to make it a string template.